site/src/pages/GroupsPage/SettingsGroupPageView.tsx

```diff
@@ -100,4 +163,21 @@ export const SettingsGroupPageView: React.FC<SettingsGroupPageViewProps> = ({
   )
 }
 
+const useStyles = makeStyles((theme) => ({
+  "@global": {
+    "em-emoji-picker": {
+      "--rgb-background": theme.palette.background.paper,
+      "--rgb-input": colors.gray[17],
+      "--rgb-color": colors.gray[4],
+    },
```

Comment:
Do we want to add this `site/src/theme/overrides.ts`? I feel like I might not look for it here.

Reply:
I just copied this from the template settings one, but I assume we'll abstract this (maybe once it's used once more?).

Reply:
I could be wrong, but I thought the `@global` key meant any classes defined within would be defined globally. So, if this is already declared in template settings, I'm surprised we need it here, too.

I'm suggesting we move it to `overrides` because we have a bunch of other globals there, and it's nice to have them colocated.

Reply:
I believe the `@global` will go away when the component is unmounted, so it won't actually apply globally to all pages, just to the global scope on this page.

Reply:
Ohhh, interesting; thanks for explaining. Alrighty, let's keep it!